uyuni/common-libs/common/__init__.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2009--2017 Red Hat, Inc.
+# Copyright (c) 2019 SUSE LLC
```

Comment:
I think we should keep both copyrights (in the end this is no a full rewrite).

Reply:
For me this is a new file. And this file has just the copyright header. I think this is ok. There is no code in this file.

Reply:
Ah, right. This is only `__init__.py`.